src/components/ReportActionItem/MoneyRequestReportPreview/MoneyRequestReportPreviewContent.tsx

```diff
@@ -0,0 +1,666 @@
+                                        }}
+                                        paymentMethodDropdownAnchorAlignment={{
+                                            horizontal: CONST.MODAL.ANCHOR_ORIGIN_HORIZONTAL.RIGHT,
+                                            vertical: CONST.MODAL.ANCHOR_ORIGIN_VERTICAL.BOTTOM,
+                                        }}
+                                        isDisabled={isOffline && !canAllowSettlement}
+                                        isLoading={!isOffline && !canAllowSettlement}
+                                    />
+                                )}
+                                {!!shouldShowExportIntegrationButton && !shouldShowSettlementButton && shouldShowRBR && (
+                                    <ExportWithDropdownMenu
+                                        policy={policy}
+                                        report={iouReport}
+                                        connectionName={connectedIntegration}
+                                        wrapperStyle={[buttonMaxWidth, styles.flexReset]}
+                                        dropdownAnchorAlignment={{
+                                            horizontal: CONST.MODAL.ANCHOR_ORIGIN_HORIZONTAL.RIGHT,
+                                            vertical: CONST.MODAL.ANCHOR_ORIGIN_VERTICAL.BOTTOM,
+                                        }}
+                                    />
+                                )}
+                                {!shouldShowSubmitButton && shouldShowRBR && (shouldShowSettlementButton || !!shouldShowExportIntegrationButton) && (
+                                    <ReviewRequiredButton
+                                        style={buttonMaxWidth}
+                                        formattedAmount={shouldShowSettlementButton ? getSettlementAmount() : undefined}
+                                    />
+                                )}
+                                {shouldShowSubmitButton && (
+                                    <Button
+                                        success={isWaitingForSubmissionFromCurrentUser}
+                                        text={translate('common.submit')}
+                                        style={buttonMaxWidth}
+                                        onPress={() => iouReport && submitReport(iouReport)}
+                                        isDisabled={shouldDisableSubmitButton}
+                                    />
+                                )}
+                            </View>
+                        </View>
+                    </View>
+                </PressableWithoutFeedback>
+            </View>
+            <DelegateNoAccessModal
+                isNoDelegateAccessMenuVisible={isNoDelegateAccessMenuVisible}
+                onClose={() => setIsNoDelegateAccessMenuVisible(false)}
+            />
+
+            {isHoldMenuVisible && !!iouReport && requestType !== undefined && (
```

Comment:
Small suggestion for consistency 
```suggestion
            {isHoldMenuVisible && !!iouReport && !!requestType && (
```

Reply:
I think it should work, thanks!